executor/src/witgen/mod.rs

```diff
@@ -375,25 +384,18 @@ impl<'a, T: FieldElement> FixedData<'a, T> {
                     poly.array_elements()
                         .map(|(name, poly_id)| {
                             let external_values = external_witness_values.remove(name.as_str());
-                            if let Some(external_values) = &external_values {
-                                if external_values.len() != poly.degree.unwrap() as usize {
-                                    log::debug!(
-                                        "External witness values for column {} were only partially provided \
-                                        (length is {} but the degree is {})",
-                                        name,
-                                        external_values.len(),
-                                        poly.degree.unwrap()
-                                    );
-                                }
-                            }
```

Comment:
We can't really check this easily anymore with dynamic sizes, and it was only a debug log anyway.